frontend/src/__tests__/cypress/cypress/support/commands/application.ts

```diff
@@ -148,31 +152,34 @@ declare global {
   }
 }
 
-Cypress.Commands.add('visitWithLogin', (url, user = HTPASSWD_CLUSTER_ADMIN_USER) => {
+Cypress.Commands.add('visitWithLogin', (relativeUrl, credentials = HTPASSWD_CLUSTER_ADMIN_USER) => {
   if (Cypress.env('MOCK')) {
-    cy.visit(url);
+    cy.visit(relativeUrl);
   } else {
-    cy.intercept('GET', url, { log: false }).as('visitWithLogin');
-
-    cy.visit(url, { failOnStatusCode: false });
-
-    cy.wait('@visitWithLogin', { log: false }).then((interception) => {
-      if (interception.response?.statusCode === 403) {
-        cy.log('Do login');
-        // do login
+    let fullUrl: string;
```

Comment:
Can we also add some logging here of the full constructued URL?

Something like 
cy.log(`Constructed full URL: ${fullUrl}`);

Reply:
It's already there on line 165:
`cy.step(`Go to: ${fullUrl}`);`

Reply:
Okay, that should be fine then.